tests/distributions/conftest.py

```diff
@@ -183,6 +183,26 @@
             prec=0.08,
             is_discrete=True),
     Fixture(pyro_dist=(dist.categorical, Categorical),
+            scipy_dist=sp.multinomial,
```

Comment:
I am not sure if the scipy multinomial will work with the non one-hot test data as is, so we will probably need to remove `scipy_dist` here and `xskip` all tests that need a scipy distribution to be specified. Alternatively, we can remove this fixture since we are doing a lot of these tests separately in `test_categorical.py`, and refactor this part later.

Reply:
Done.

Reply:
(scipy.multinomial works with one-hot but not non-one-hot, so I've removed if from the non-one-hot fixture)